src/app/navigation/examples/application-launcher-example.component.html

```diff
@@ -0,0 +1,61 @@
+<ul class="nav navbar-nav">
+</ul>
+<ul class="nav navbar-nav navbar-right navbar-iconic">
+    <li class="dropdown">
```

Comment:
Clicking on the items in the app launcher opens a new window and leaves the demo in a funky state -- the welcome page is blank.

It appears to be related to the Url used to launch the new window. The error in the browser console is:

Error: Uncaught (in promise): Error: Cannot match any routes. URL Segment: 'ipsum/intellegam/suavitate'

![screen shot 2018-01-25 at 5 44 46 pm](https://user-images.githubusercontent.com/17481322/35416546-791f5714-01f7-11e8-9f7f-766c39bd499c.png)